example/ios/Demo/Permissions/BluetoothPermissionViewController.swift

```diff
@@ -18,15 +18,11 @@ class BluetoothPermissionViewController: UITableViewController {
         switch (indexPath.row) {
         case 1:
             debugPrint(bluetoothManager.checkPermit().name)
-            break
         case 2:
             debugPrint(bluetoothManager.checkSupport().name)
-            break
         case 3:
             bluetoothManager.openSettings()
-            break
         default:
-            break
```

Comment:
Okay, `break` is not required in Swift but for cases which you want to ignore (in this case default case) you have to use `break` still